addons/account/tests/test_fiscal_position.py

```diff
@@ -154,3 +154,96 @@ def test_20_fp_one_tax_2m(self):
         mapped_taxes = self.fp2m.map_tax(self.src_tax)
 
         self.assertEqual(mapped_taxes, self.dst1_tax | self.dst2_tax)
+
+    def test_30_fp_delivery_address(self):
+        # Make sure the billing company is from Belgium (within the EU)
+        self.env.company.vat = 'BE0477472701'
+        self.env.company.country_id = self.be.id
+
+        # Reset any existing FP
+        self.fp.search([]).write({'auto_apply': False})
```

Comment:
No need to change for now but FYI, you can make it that way too:
```suggestion
        self.fp.search([]).auto_apply = False
```

Reply:
Good to know, looks cleaner indeed :slightly_smiling_face: